.github/workflows/on-push-master.yml

```
@@ -102,6 +102,30 @@ jobs:
       GP2_CONTENTFUL_WEBHOOK_AUTHENTICATION_TOKEN: ${{ secrets.GP2_CONTENTFUL_WEBHOOK_AUTHENTICATION_TOKEN }}
       SLACK_WEBHOOK: 'n-a'
 
+  empty-integration-test-env-contentful-migration:
```

I think I would prefer to stick to the similar naming convention of that within the reusable-deployment, something like:
```suggestion
  crn-contentful-migrate-empty:
```